src/ffi.rs

```diff
@@ -63,13 +65,15 @@ pub unsafe extern "C" fn install(
 
         let schemes = (CStr::from_ptr(schemes).to_str()?).to_owned();
 
-        rust_install(
+        let res = rust_install(
```

Comment:
Same thing here: it should be `rust_install(...)?` and return `Ok(())` on the last line of the function